### ISSUE TITLE
Using yolov4 instead of v3

### ISSUE THREAD

**Issue by utkutpcgl** [theAIGuysCode/yolov3_deepsort#9]
Recently the new version came out. Is there an easy way to integrate it with the code?

**Comment by vihari1729** [comment]
@utkutpcgl  Did you find any way integrate yolov4model(which has been trained on dataset) for tracking?

**Comment by utkutpcgl** [comment]
@vihari1729. I modified an existing github repository of yolov4 [https://github.com/hunglc007/tensorflow-yolov4-tflite] , implemented in tf. After adapting the model function it worked with @theAIGuysCode yolov3deepsort.

**Comment by vihari1729** [comment]
@utkutpcgl Thanks, It is working now

**Comment by MuhammadAsadJaved** [comment]
@utkutpcgl  Hi,  How about the speed of YOLOV4+DeepSort?   + Have you tried Yolov4-Tiny + DeepSort as well?

**Comment by utkutpcgl** [comment]
Hello @MuhammadAsadJaved. I did not try it with tiny. I tried it with v3 and v3 tiny. V4 was ~25% more successful than v3 in terms of flickering etc. However, v3-tiny could not match my expectations. 
I trained it on 1070ti with 2000 epoch. The batch size was 32 or 64, I do not remember exactly. I had very few sample images around 100 or so. I expected tiny to learn faster and generalize better, however, it ended up as v4 being the best. 
I cannot really comment on v4 tiny. 
Hopefully this helps you.
